### PR TITLE
Improve look and feel

### PR DESCRIPTION
**Pull request by vie10**
The content paper:

- Added minHeight for fill the visible space
- Added padding for beautify look and feel

Other:

- Added new size constant for padding

### CODE REVIEW

File: src/main/kotlin/team/karakum/common/Sizes.kt [hunk 1]
```diff
@@ -10,4 +10,8 @@ object Sizes {
     object Sidebar {
         val Width = 135.px
     }
+
+    object Padding {
+        val Graceful = 30.px
```
Comment (vie10) [inline src/main/kotlin/team/karakum/common/Sizes.kt:15]: Used in 2 places so is useful

File: src/main/kotlin/team/karakum/component/Content.kt [hunk 2]
```diff
@@ -28,10 +32,14 @@ val Content = FC<Props> {
                 sx = jso {
                     flexGrow = FlexGrow(1.0)
                     marginTop = Header.Height
-                    padding = 30.px
+                    padding = Padding.Graceful
                 }
 
                 Paper {
+                    sx = jso {
```
Comment (vie10) [inline src/main/kotlin/team/karakum/component/Content.kt:39]: Twice minus padding graceful because of the paper is padded by bottom and top

**Comment by vie10** [comment]
Screenshot 2022-02-06 at 14 24 23
Look and feel after improves. LGTM

File: src/main/kotlin/team/karakum/component/Content.kt [hunk 3]
```diff
@@ -58,3 +66,15 @@ val Content = FC<Props> {
         }
     }
 }
+
+private val visibleHeight: Length
```
Comment (aerialist7) [inline src/main/kotlin/team/karakum/component/Content.kt:70]: Think this calculations are redundant.

Now I've added Grid layout. 
Try to tune the grid to get `Content` component background to fill the entire grid cell

**Review by aerialist7 — CHANGES_REQUESTED** [review]

**Comment by aerialist7** [comment]
@vie10, also let me give a recommendation to you: please create a new branch in your repo and give it a PR-related name.
Then do your commits. And then send a PR.

This approach is more convenient and do not produce redundant commits like these
image

**Comment by vie10** [comment]
> @vie10, also let me give a recommendation to you: please create a new branch in your repo and give it a PR-related name.
> 
> Then do your commits. And then send a PR.
> 
> 
> 
> This approach is more convenient and do not produce redundant commits like these
> 
> image
> 
> 

Thx, I'll

File: src/main/kotlin/team/karakum/common/Sizes.kt (hunk 1 above)
Comment (aerialist7) [reply src/main/kotlin/team/karakum/common/Sizes.kt:15]: Yep, but let's call it `Default`

**Review by aerialist7 — CHANGES_REQUESTED** [review]
Small refactoring required

**Review by aerialist7 — APPROVED** [review]

**Pull request merged**